### PR TITLE
appium-server 0.0.1

### PR DESCRIPTION
**Pull request by moatazeldebsy**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/appium-server/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- react-native
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
```
Comment (koral--) [inline steps/appium-server/0.0.1/step.yml:28]: None of those are used in script but npm is OTOH.

File: steps/appium-server/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+title: Appium Server
```
Comment (godrei) [inline steps/appium-server/0.0.1/step.yml:1]: `Run Appium Server` would be a better title: https://devcenter.bitrise.io/contributors/create-your-own-step/#the-title

File: steps/appium-server/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- appium_version: Default Value - you can leave this empty if you want to
```
Comment (godrei) [inline steps/appium-server/0.0.1/step.yml:37]: One of the value options should be the default value (`v1.21.0` or `v1.20.2`) for the input.

**Review by godrei — CHANGES_REQUESTED** [review]
@moatazeldebsy thanks for the new Step, please review my comments.

This is how you can fix the issues: https://devcenter.bitrise.io/contributors/sharing-steps-with-all-bitrise-users/#fixing-issues-in-a-steplib-pull-request

**Pull request closed without merging**